_includes/specials/taproot/en/03-p2wpkh-to-p2tr.md

```diff
@@ -0,0 +1,119 @@
+For wallets that already support receiving and spending v0 segwit P2WPKH
+outputs, upgrading to v1 segwit P2TR for single sig should be easy.
+Here are the main steps:
+
+- **Use a new BIP32 key derivation path:** you don't need to change your
```

Comment:
I missed this in the other 3 of this series, but we lose the bulleted list anchor links when embedding

Reply:
Good catch!  I'm not sure why it doesn't work and I can't think of a trivial fix for it, so I opened a new issue.  I'll look into it ASAP.

Reply:
Marking this as resolved now that we have an issue tracking it.